### PR TITLE
[FIX] im_livechat, *: Fix broken test page of chatbot

### PR DESCRIPTION
**Pull request by alexkuhn**
*: website_livechat

Task-3006246
Task-3010911

### CODE REVIEW

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

File: addons/website_livechat/__manifest__.py [hunk 1]
```diff
@@ -37,7 +37,6 @@
         ],
         'web.assets_frontend': [
             'website_livechat/static/src/legacy/public_livechat.js',
-            'website_livechat/static/src/legacy/website_livechat_chatbot_test_script.js',
```
Comment (alexkuhn) [inline addons/website_livechat/__manifest__.py:40]: Just forgot to remove in the manifest.
Made runbot red, but for some reasons had no issue locally even with `-u`

**Comment by alexkuhn** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**